zebra-rpc/src/methods.rs

```diff
@@ -1439,6 +1444,10 @@ pub enum GetRawTransaction {
         /// The height of the block that contains the transaction, or -1 if
         /// not applicable.
         height: i32,
+        /// The confirmations of the block that contains the transaction,
+        /// or None if the transaction is in the mempool.
+        #[serde(skip_serializing_if = "Option::is_none")]
+        confirmations: Option<u32>,
```

Comment:
`zcashd` always provides this field, but uses  `0` for mempool transactions:
https://github.com/zcash/zcash/blob/c0fbeb809bf2303e30acef0d2b74db11e9177427/src/rpc/rawtransaction.cpp#L332

Reply:
Whoops, I missed this, thank you. Fixed in [`bb50243`](https://github.com/ZcashFoundation/zebra/pull/6287/commits/0b5c7bfc4c1a75ce3953f947738a3d830e073eb8).